bridge/engine/EngineMixer.h

```diff
@@ -205,22 +207,45 @@ class EngineMixer : public transport::DataReceiver
               _transport(transport),
               _extendedSequenceNumber(extendedSequenceNumber)
         {
+            lockOwner();
         }
 
-        PacketT* _packet;
-        AllocatorT* _allocator;
-        transport::RtcTransport* _transport;
-        uint32_t _extendedSequenceNumber;
+        IncomingPacketAggregate(IncomingPacketAggregate&& rhs)
```

Comment:
Nice, I will try this. Then it will move the responsibility or releaseing job count when pushing to the queue. The lie time of the IcomingAggregate will then be the life time of the ref increment.